xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/mockData.js

```diff
@@ -0,0 +1,157 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+
+// This is mock data used while the front-end is not communicating with the macros
+// The data structure is not definitive, and may change to match the final specs
+
+define({
+
+  query: {
+    properties: ["doc_title", "age", "job", "other"],
+
+    source: {
+      id: "...",
+      url: "...",
+    },
+
+    hiddenFilters: {},
+
+    filters: {},
+
+    sort: [ {properties: "age", descending: true} ],
+
+    offset: 0,
+    limit: 10,
+
+    layouts: ["table", "cards"],
+    defaultLayout: "table",
```

Comment:
I think this needs to go in **meta** because the layout doesn't influence the data or how the data is fetched from the database on the server. The layout controls how the data is displayed so for me it naturally goes inside **meta**. ``properties`` vs. ``propertyDescriptors`` is different, because you need the list of properties on the server side when fetching the data. I don't see any problem having both ``layouts`` and ``layoutDescriptors`` in **meta**. We need both later when from design mode you may want to allow the user to choose the layouts that should be available to the end user.